packages/salesforcedx-vscode-core/package.json

```diff
@@ -29,7 +29,7 @@
     "@salesforce/salesforcedx-sobjects-faux-generator": "51.9.0",
     "@salesforce/salesforcedx-utils-vscode": "51.9.0",
     "@salesforce/schemas": "^1",
-    "@salesforce/source-deploy-retrieve": "1.1.20",
+    "@salesforce/source-deploy-retrieve": "2.0.0",
```

Comment:
FYI, `@salesforce/source-deploy-retrieve` is also used in the `salesforcedx-vscode-apex` module.

Reply:
@lcampos I'll add a follow-up story for updating apex. It probably isn't wise to have two different versions in the extensions.